docs/public/operators-on-minikube.adoc

```diff
@@ -0,0 +1,204 @@
++
+[source,sh]
+----
+$ git clone https://github.com/redhat-developer/service-binding-operator/
+$ cd service-binding-operator
+$ git fetch --tags
+$ git checkout tags/v0.1.1-364 -b v0.1.1-364
+$ cd deploy
+----
+
+. Next, install its CRD (link:https://docs.openshift.com/container-platform/latest/operators/understanding/crds/crd-extending-api-with-crds.html[Custom Resource Definition]) and create link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/[ServiceAccount], https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole[Role] and link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding[RoleBinding] it requires:
++
+[source,sh]
+----
+$ kubectl create -f crds/apps.openshift.io_servicebindingrequests_crd.yaml
+$ kubectl create -f service_account.yaml -f role.yaml -f role_binding.yaml
+----
+
+. Now, create a link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole[ClusterRole] and a link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding[ClusterRoleBinding] that are required to allow the Service Binding Operator access resources of API groups `apps.openshift.io` and `apiextensions.k8s.io` at cluster level:
++
+[source,sh]
+----
+$ kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sbo-cluster-role
+rules:
+- apiGroups: 
+  - apps.openshift.io
+  resources: 
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
+  verbs:
+  - get
+  - list
+  - patch
+  - watch
+EOF
+
+$ kubectl create -f - << EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sbo-cluster-role-binding
+subjects:
+- kind: ServiceAccount
+  name: service-binding-operator
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: sbo-cluster-role
+  apiGroup: rbac.authorization.k8s.io
+EOF
+----
+
+. Next, we modify the `operator.yaml` file to install the Operator in a specific namespace and use the container image that matches the version of the git commit (`git log -n1 --oneline`) of the repository. Make change as per below `git diff` but make sure to set the value of `namespace` to the namespace where you want to install the Operator:
++
+[source,diff]
+----
+diff --git a/deploy/operator.yaml b/deploy/operator.yaml
+index 2cd90ec7..21c4768c 100644
+--- a/deploy/operator.yaml
++++ b/deploy/operator.yaml
+@@ -4,0 +5 @@ metadata:
++  namespace: default
+@@ -19 +20 @@ spec:
+-          image: REPLACE_IMAGE
++          image: quay.io/redhat-developer/app-binding-operator:0799ba7
+----
+
+. Finally, install the Operator using below command:
++
+[source,sh]
+----
+$ kubectl create -f operator.yaml
```

Comment:
could we create one liner from this? 
```
curl -sL https://raw.githubusercontent.com/redhat-developer/service-binding-operator/v0.1.1-364/deploy/operator.yaml | \ 
  sed "s/REPLACE_IMAGE/quay.io\/redhat-developer\/app-binding-operator:0799ba7/" | \
  kubectl create -f -
```

Reply:
Why not? I had assumed otherwise. However, what about Windows users? I'm not sure if this would work on Powershell.